sharktank/sharktank/models/llm/llm.py

```diff
@@ -362,6 +369,13 @@ def __init__(
                 fake_quant=fake_quant,
                 softcap=config.hp.attention_softcap,
                 model_arch=config.hp.model_arch,
+                use_rope=use_rope,
+                use_qk_norm=block_index in config.rope_layers and config.use_qk_norm
+                if config.rope_layers
+                else False,
```

Comment:
Could you prepare this arguments before the call? Lets not put if-else at the call site. People like clean calls.